src/rt/sched/cown.h

```diff
@@ -113,8 +123,8 @@ namespace verona::rt
      **/
     std::atomic<size_t> weak_count = 1;
 
-    std::atomic<Status> status{};
     std::atomic<uintptr_t> bp_state{(Cown*)nullptr | Priority::Normal};
+    bool has_token = false;
```

Comment:
Can you leave a comment to pack this into something else?  Seems pretty wasteful on space.

Reply:
Comment added.